scripts/clang-tidy-only-diff.sh

```diff
@@ -4,16 +4,9 @@
 set -o errexit -o pipefail -o nounset
 
 readonly base=$(git merge-base refs/remotes/origin/master HEAD)
-readonly build_dir=build
-
-readonly CLANG_TIDY_VERSION=7.0.0
+readonly build_dir=${1:-build}
```

Comment:
make this a command line argument. my setup says `build/{build_type}`, e..g `build/Release`

Reply:
do note, this replaces the previous argument to replace `nproc`. it was only used by CI in the code base, where I removed it too. could make it a second argument to not break anyone's personal setup though?!